### ISSUE TITLE
Can't import curses on pypy3.5-5.9-beta-linux_x86_64-portable

### ISSUE THREAD

**Issue by eli-b** [squeaky-pl/portable-pypy#62]
```
$ pypy3.5-5.9-beta-linux_x86_64-portable/bin/pypy -m curses
Traceback (most recent call last):
  File "/home/vagrant/pypy3.5-5.9-beta-linux_x86_64-portable/lib-python/3/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/home/vagrant/pypy3.5-5.9-beta-linux_x86_64-portable/lib-python/3/runpy.py", line 142, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/home/vagrant/pypy3.5-5.9-beta-linux_x86_64-portable/lib-python/3/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/home/vagrant/pypy3.5-5.9-beta-linux_x86_64-portable/lib-python/3/curses/__init__.py", line 13, in <module>
    from _curses import *
  File "/home/vagrant/pypy3.5-5.9-beta-linux_x86_64-portable/lib_pypy/_curses.py", line 9, in <module>
    from _curses_cffi import ffi, lib
ImportError: /home/vagrant/pypy3.5-5.9-beta-linux_x86_64-portable/lib_pypy/_curses_cffi.pypy3-59-x86_64-linux-gnu.so: undefined symbol: unget_wch
```
Version 5.8 runs this without issue.

**Comment by squeaky-pl** [comment]
This is because of bundled ncurses missing wide character support. Upcoming 5.10 version will contain a fix.

**Comment by squeaky-pl** [comment]
Fixed in 5.10
